### ISSUE TITLE
Python new versions compatibility

### ISSUE THREAD

**Issue by rafaelpsilva07** [saullocastro/compmech#20]
Python new versions compatibility
--------------------------------------
The last version of **compmech** is compatible with 2.7, 3.4 and 3.5.
I'd like to propose a version update of this library so we can use more modern versions of the interpreter and also jupyter kernels.

I'm not sure if changing the setup.py file is enough, I have a hard time building libraries that has compiled languages (like C and Fortran).

Alternative solution
----------------------
An alternative solution to update this library, is adding new functionalities to panels library, which is already compatible with python newer versions.

**Comment by saullocastro** [comment]
That's a good suggestion and I believe that the effort is worth the benefits. Furthermore, many functionalities here will not be moved to the panels module.

I'll be able to work on it after the 4th of October.

**Comment by rafaelpsilva07** [comment]
I'd like to bring some updates on this issue.

I've had some problems building ```compmech``` with Python 3.10. There were some deprecation warning concerning ```numpy.distutils``` and also some issues with collections while running the tests.

On the other hand, the **good news** is that the package works perfectly until Python 3.9 (I've just build with this distro in order to run my jupyter notebooks and tests results remained the same). I believe the only modification needed to the next built is the **metadata** of the Pytyhon 3.8 and 3.9. The ```pip``` takes these metadatas into consideration when downloading from PYPI.org.

To summarize, I think we just need to add these metadata before the new release and we can close this issue for now.

![image](https://user-images.githubusercontent.com/46754669/197063507-ab6734aa-ccb0-44bf-a2a9-db4193d7dd68.png)

**Comment by rafaelpsilva07** [comment]
Current master branch is already working with python new veresions. I'm using compmech with Python 3.10.
So this problem should be solved within the next deploy.
